frame/dapps-staking/src/migrations.rs

```diff
@@ -353,3 +353,31 @@ pub mod v2 {
         Ok(())
     }
 }
+
+pub mod v3 {
```

Comment:
This isn't v3 so no need to have it here. Also, it isn't really a migration.

I'd recommend you just do this in an struct implementing the OnRuntimeUpgrade trait.

Reply:
You should remove this file from the PR.